src/views/Home/components/NewsCard/index.js

```diff
@@ -29,17 +29,17 @@ export default class DailyNews extends Component {
   }
 
   componentWillMount() {
-   this.loadNews();
+    this.loadNews();
   }
 
   // loads the news (not currently "by category")
   async loadNews() {
     // let newsCategories = await NewsService.getCategories();
     let todaysNews = await NewsService.getTodaysNews();
 
-    this.setState({ 
-      // newsCategories: newsCategories, 
-      news: todaysNews
+    this.setState({
+      // newsCategories: newsCategories,
+      news: todaysNews,
```

Comment:
Yes, it's common practice in JavaScript to end arrays, objects, etc. with a trailing comma (you can actually see in the first file changed ".prettierrc.json" that our linter is set to require that). The reasoning is that, since you don't have to add a comma to existing lines when you make an update, git can detect the difference more accurately. Notice, for example, that git has two lines marked as different in .prettierrc.json because that file doesn't have trailing commas, so the last line from the old file has a comma added, and then the new line is also added.